src/helpers/paginate.helper.js

```diff
@@ -0,0 +1,12 @@
+/**
+*  this will  help to separate and content into discrete pages
```

Comment:
@mujohn26 Kindly update this description and provide a meaningful one. For instance, this description lets one understand that this function is given contents and it will return multiple pages but that's far away from what this function does.

Reply:
I changed the message